### PR TITLE
Fix: Typo for diligence,distributed,documenting,controlled

### LINKED ISSUE #701: [Bug]: fix spelling issues
We spellcheck the content using `make spellcheck`, which uses a dictionary of words `scripts/cspell/project-words.txt`. However, this dictionary was seeded with some clearly misspelled words, at minimum those shown in the diff below. The goal is to fix the occurrences of those misspelled words in the main content (e.g. by finding them with `grep -rn {word} content/`) and remove them from the dictionary file.

```diff
--- a/scripts/cspell/project-words.txt
+++ b/scripts/cspell/project-words.txt
@@ -6,15 +6,12 @@ ashmita
 autoscale
 autoscaler
 bansal
-behaviour
 bohara
 bootcamp
-boths
 certfile
 changeme
 clickhouse
 cncf
-conrolled
 containerd
 cqlsh
 cqlshrc
@@ -22,9 +19,6 @@ daemonset
 datacenter
 datasource
 developercertificate
-dilligence
-distirbuted
-documentiing
 downsampling
 dropwizard
 dsoie
@@ -51,8 +45,6 @@ htpasswd
 ingester
 ingester's
 ingesters
-ingestor
-initialisation
 istio
 istio's
 jaegertracing
@@ -128,18 +120,15 @@ statefulset
 strimzi's
 struct
 tchannel
-telemtery
 thriftrw
 tolerations
 tracegen
-trft
 uberctx
 unmarshaling
 upsample
 usercert
 userkey
 valuen
-variery
 vlog
 volz
 weaveworks
```

### PR DESCRIPTION
**Pull request by JeevaRamanathan**
## Which problem is this PR solving?
- Resolves #701 

## Checklist
- [x] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [x] I have signed all commits

### CODE REVIEW

File: scripts/cspell/project-words.txt [hunk 1]
```diff
@@ -14,7 +14,7 @@ certfile
 changeme
 clickhouse
 cncf
-conrolled
+controlled
```
Comment (yurishkuro) [inline scripts/cspell/project-words.txt:17]: Are these words even necessary in the dictionary?

Comment (JeevaRamanathan) [reply scripts/cspell/project-words.txt:17]: Not sure, can you please suggest either those words [diligence,distributed,documenting,controlled] should be removed from this file or leavr untouched?

Comment (yurishkuro) [reply scripts/cspell/project-words.txt:17]: If the linter is successful without having these words in the dictionary we should remove them.

Comment (JeevaRamanathan) [reply scripts/cspell/project-words.txt]: I have reorder that list alphabetically due to which it was failing previously.  Can you please check

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**